### DISCUSSION TITLE
Has anyone else noticed emails are only sent with Content-Type: text/plain; charset="utf-8" using Buttondown's new writing interface?

### DISCUSSION THREAD

**Discussion by JayCuthrell** [buttondown/discussions#2] [Q&A] [answered]
Has anyone else noticed emails are only sent with `Content-Type: text/plain; charset="utf-8"` using Buttondown's new writing interface?

Results:

- Email appears as plain text HTML
- Email appears as plain text Markdown
- Email appears as combination of plain text HTML and Markdown

Am I holding it wrong? 😄 

Inbox view

<img width="494" alt="Screenshot 2022-12-03 at 12 18 06 PM" src="https://user-images.githubusercontent.com/95317/205453350-ed28fbe2-1d0a-458e-8a0c-b761c0a089b2.png">

Headers view

<img width="811" alt="Screenshot 2022-12-03 at 12 19 50 PM" src="https://user-images.githubusercontent.com/95317/205453422-7b97cbd5-030d-4d60-afad-2a8b435fe444.png">

**Comment by jmduke** [comment, marked as answer]
Aha, mind confirming that this was due to the `plaintext-only` setting?

**Reply by JayCuthrell** [reply]
Correct. A uniquely early adopter quirk. 😄
